layouts/speaker/single.html

```diff
@@ -72,6 +72,30 @@ <h2 class="speaker-page">{{ .Title }}</h2>
               {{- end -}} <!-- end if eq $talk_slug $event_slug -->
             {{- end -}} <!-- end range where $.Site.Pages "Type" "speaker" -->
           </ul>
+    <br>
+      <h3>Workshops by {{ .Title }}</h3>
+           <ul class="list-group list-group-flush">
+             {{ $.Scratch.Set "speaker" .File.BaseFileName }}
+             {{- range where $.Site.Pages "Type" "workshop" -}}
+               {{- $talk_path := split .File.Path .Site.Params.PathSeparator -}}
+               {{- $talk_slug := index $talk_path 1 -}}
+               {{- if eq $talk_slug $event_slug -}}
+                 <!-- Now we can display stuff! -->
+                 {{ range .Params.speakers }}
+                   {{ if eq . ($.Scratch.Get "speaker") }}
+                     {{ $.Scratch.Set "display" "true" }}
```

Comment:
Then here if $.Scratch.Get "speaker_headline" is 'false" but "display" is "true" you show the headline stuff and open the <ul> element, and then set "speaker_headline" to false.